test/users.js

```diff
@@ -0,0 +1,90 @@
+const chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  dictum = require('dictum.js'),
+  server = require('../app'),
+  should = chai.should();
+
+chai.use(chaiHttp);
+describe('/users POST', () => {
+  it('should fail because email is missing', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
```

Comment:
dont delegate error handling to the database, do it before it even reaches the controller logic, otherwise you may do a bunch of calculations before having it fail due to a saving error while you could have just validated it beforehand and avoided useless processing